client/src/components/Amenities/Amenities.js

```diff
@@ -0,0 +1,126 @@
+import React, { Component } from 'react';
+import FontAwesome from 'react-fontawesome';
+import PopUp from './../Amenities/PopUp.js';
+import ReactModal from 'react-modal';
+import Icons from './Icons.js';
+
+ReactModal.setAppElement('#root');
+
+class Amenities extends Component {
+  constructor(props) {
+    super(props);
+    console.log('amenities here', this.props.amenities);
+    this.state = {
+      showPopUp: false
```

Comment:
Perhaps isPopUpDisplayed is a better name- showPopUp sounds like a function not a boolean